src/Client/Web/Bit.Client.Web.BlazorUI/Components/DropDown/BitDropDown.Fluent.scss

```diff
@@ -289,6 +289,41 @@
     }
 }
 
+.overflow-hidden {
+    overflow: hidden !important;
```

Comment:
We must never use `!important` in our codes.
plz remove this keyword and if there is some issue with removing it, you should find another way to solve it. using `!important` is not the solution :)